academy-of-management-review.csl

```diff
@@ -234,16 +252,23 @@
           <group>
             <text macro="container-contributors"/>
             <text macro="secondary-contributors"/>
-            <text variable="container-title" font-style="italic" font-weight="bold"/>
+            <choose>
+              <if type="article-journal article-magazine article-newspaper paper-conference" match="any">
```

Comment:
For this journal, most conference proceedings that are cited will be published in journal-like format (specifically the Academy of Management Proceedings), so the proceedings title should be capitalized like a journal title. I'm not sure the best way to handle both formats of proceedings correctly (as book-like proceedings should be in Title case). Would it work to test for an editor (as most journal-like proceedings don't have editors that are cited)?